rust_language/resolve.go

```diff
@@ -131,6 +135,14 @@ func (l *rustLang) Resolve(c *config.Config, ix *resolve.RuleIndex,
 			}
 		}
 
+		if ruleData.dependsOnBuildScript {
+			build_script_label, err := label.Parse(":build_script")
```

Comment:
I think it would be cleaner to replace:
```go
dependsOnBuildScript bool
```
with:
```go
buildScript *label.Label
```

That way the resolve stage doesn't have a fallible label construction, and it will be easier to extend to support build scripts for pure-bazel workflows (with potentially arbitrary labels) in the future.